src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -74,23 +74,39 @@ def load_arguments(self, _):
 
         # keyCredentials
         c.argument('start_date', arg_group='keyCredentials',
-                   help="Date or datetime at which credentials become valid(e.g. '2017-01-01T01:00:00+00:00' or '2017-01-01'). Default value is current time")
+                   help="Date or datetime at which credentials become valid (e.g. '2017-01-01T01:00:00+00:00' or "
+                        "'2017-01-01'). Default value is current time")
         c.argument('end_date', arg_group='keyCredentials',
-                   help="Date or datetime after which credentials expire(e.g. '2017-12-31T11:59:59+00:00' or '2017-12-31'). Default value is one year after current time")
+                   help="Date or datetime after which credentials expire (e.g. '2017-12-31T11:59:59+00:00' or "
+                        "'2017-12-31'). Default value is one year after current time")
         c.argument('key_value', arg_group='keyCredentials',
                    help='the value for the key credentials associated with the application')
         c.argument('key_type', arg_group='keyCredentials',
-                   help='the type of the key credentials associated with the application', arg_type=get_enum_type(['AsymmetricX509Cert', 'Password', 'Symmetric'], default='AsymmetricX509Cert'))
+                   help='the type of the key credentials associated with the application',
+                   arg_type=get_enum_type(['AsymmetricX509Cert', 'Password', 'Symmetric'],
+                                          default='AsymmetricX509Cert'))
         c.argument('key_usage', arg_group='keyCredentials',
-                   help='the usage of the key credentials associated with the application.', arg_type=get_enum_type(['Sign', 'Verify'], default='Verify'))
-        c.argument('credential_description', help="the description of the password")
```

Comment:
`--credential-description` is dropped according to https://github.com/Azure/azure-cli/issues/20561.